src/Service.Tests/SqlTests/RestApiTests/Insert/MsSqlInsertApiTests.cs

```diff
@@ -302,6 +314,57 @@ public async Task InsertOneInViewBadRequestTest()
             );
         }
 
+        /// <summary>
+        /// Test to validate that even when an insert DML trigger is enabled on a table, we still return the
+        /// latest data as it is present after the trigger gets executed. Whenever an insert DML trigger is enabled,
+        /// we use a subsequent SELECT query to get the data instead of using OUTPUT clause.
+        /// </summary>
+        [TestMethod]
+        public async Task InsertOneInTableWithInsertTrigger()
+        {
+            // Validate that insert operation succeeds when an insert DML trigger is enabled for a table
+            // with autogenerated primary key. The trigger updates the salary to max(0,min(100,salary)).
+            string requestBody = @"
+            {
+                ""name"": ""Joel"",
+                ""salary"": 102
+            }";
+
+            string expectedLocationHeader = $"id/{STARTING_ID_FOR_TEST_INSERTS}/u_id/2";
```

Comment:
This test should also validate only a single record is updated when in the FTE table you have multiple records with `u_id = 2` becuase the definition in the table is `u_id int DEFAULT 2`. so for example:
id: 1 u_id=2 name=bob
id: 2 u_id=2 name janet

an update to pk (id:1, u_id2) does not also update the second record due. This validates your business logic where you add non-autogen PKs to a temp table and then join with the scope_identity value. (maybe this would only apply as a concern to the insert test.)

Reply:
This is an insert test. Also this table has a composite PK (id,u_id) which uniquely identifies a record. To make an update, we need to provide value for every PK field. So, how can mutiple records get updated?

Reply:
ok seems i posted on wrong test